EIPS/eip-1.md

```diff
@@ -95,20 +95,111 @@ A PR moving an EIP from Last Call to Final SHOULD contain no changes other than
+
+**An EIP's solution to the problems should be discussed in the Rationale, and must not be discussed in the Motivation.*
+
+This is a non-normative section.
+
+### Specification 
+
+**Required for all EIPs**
+
+The Specification describes the EIP's solution to the problems listed in the Motivation section. The specification must clearly identify the characteristics of the solution in detail. The specification should be as general as possible while avoiding ambiguity.
+
+This is a normative section. If this section conflicts with any other normative section, the other section must be amended to satisfy the specification unless it is evident that the specification is incorrect.
+
+### Rationale
+
+**Required for all EIPs**
+
+The rationale describes how the specification solves the problems described in the Rationale, and discusses why particular design decisions were made over others, including any tradeoffs made. The rationale must discuss important objections or concerns raised about the EIP.
+
+**The Rationale should assume that the reader has read the Motivation. It is okay to restate a problem and relevant characteristics when discussing the solution, but do not mention irrelevant characteristics.**
+
+This is a non-normative section.
+
+### Backwards Compatibility
+
+**Required for all EIPs**
+
+The Backwards Compatibility section describes any backward incompatibilities and their consequences. If no backward incompatibilities are found, the following wording can be used:
+
+```md
+No backward compatibility issues found.
+```
+
+If the EIP is in draft or review, the following wording should be used until significant discussion has taken place:
+
+```md
+No backward compatibility issues found. Needs discussion.
+```
+
+This is a non-normative section. Additions of newly-identified backward compatibility issues should be proposed, even after an EIP reaches final.
+
+### Test Cases
+
+**Required for all Core EIPs; Optional for all non-Core EIPs**
+
+The Test Cases section provides data with which to test implementations. Tests should either be inlined in the EIP as input/expected output pairs (recommended), or included as unit tests in the EIP's assets directory.
+
+This is a normative section, but in the case of a conflict, the Specification section is considered more normative, and the Test Cases should be amended.
+
+### Reference Implementation
+
+**Optional for all EIPs**
+
+The Reference Implementation provides a minimal non-production-ready implementation that people can use to assist in understanding or implementing the specification. More complex, production-ready implementations can be submitted to [ERCRef](https://github.com/ercref/ercref-contracts) and/or [OpenZeppelin Contracts](https://github.com/OpenZeppelin/openzeppelin-contracts).
+
+This is a non-normative section. In the case of a conflict with the Specification section, the Reference Implementation should be amended.
+
```

Comment:
In #7230 I've suggested that for Core EIPs this section requires at lease a link to the relevant commits to the Executable Specifications.